docs/source/index.rst

```diff
@@ -44,6 +44,7 @@ DAML SDK documentation
    app-dev/authentication
    app-dev/ledger-api
    app-dev/bindings-ts/index
+   json-api/index
```

Comment:
I would move this one line up. Then the bindings sit next to each other and the two APIs sit next to each other too.

Reply:
Because `app-dev/ledger-api` introduces [interacting with the ledger in general](https://github.com/digital-asset/daml/blob/2b036f596deeebfa2b0d9ab4d82b3bc5ac5485d1/docs/source/app-dev/ledger-api.rst#whats-in-the-ledger-api), this goes

1. ledger API (via whatever mechanism),
2. high-level JSON API (bindings-ts),
3. low-level JSON API (json-api/index),
4. gRPC bindings,
5. gRPC direct,
6. building your own gRPC bindings+codegen

I think this order makes sense, or maybe [2,3,1,...] makes sense (with bindings-ts borrowing some of the front matter from ledger-api), but I think bindings to JSON API should precede JSON API itself in the same way that the gRPC bindings precede the direct gRPC description. Do you think other changes need to be made to make that work?

Reply:
That makes sense. Thanks a lot for the explanation.